src/blocks/stars.css

```diff
@@ -0,0 +1,26 @@
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+  background-image: url(../images/star.svg);
+  background-repeat: no-repeat;
+  background-position: 50% 50%;
+  margin-right: 4px;
+  /* background-color: aquamarine; */
+  /* border: 1px solid black; */
```

Comment:
Do you need these comments in your final code?

Reply:
thanks! you are a good reviewer